android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -876,6 +855,42 @@ private void initializePlayerCore(ReactExoplayerView self) {
         }
     }
 
+    private AdsMediaSource initializeAds(MediaSource videoSource, Source runningSource) {
+        AdsMediaSource mediaSourceWithAds = null;
+        AdsProps adProps = source.getAdsProps();
+        if (adProps != null) {
```

Comment:
I think we can change it as shown below for simplicity.
```
AdsProps adProps = source.getAdsProps();
if (adProps == null || adProps.getAdTagUrl() == null) {
        exoPlayerView.hideAds();
        return null;
    }
     exoPlayerView.showAds();
     ...
```

Reply:
I undestand your point, but there are 3 differents cases where I want to call `exoPlayerView.showAds();` so I prefer current version